nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/CoreFeature.java

```diff
@@ -296,47 +294,51 @@ protected void batchRemoveDocuments(IterableQueryResult results) {
             }
             ids.add(new IdRef(id));
             if (ids.size() >= 100) {
-                batchRemoveDocuments(ids);
+                batchRemoveDocuments(adminSession, ids);
                 ids.clear();
             }
         }
         if (!ids.isEmpty()) {
-            batchRemoveDocuments(ids);
+            batchRemoveDocuments(adminSession, ids);
         }
     }
 
-    protected void batchRemoveDocuments(List<DocumentRef> ids) {
+    protected static void batchRemoveDocuments(CoreSession adminSession, List<DocumentRef> ids) {
```

Comment:
Why static?

Reply:
because as I don't use anything in the feature and to avoid error by confusing adminSession and session, I will remove it